validation-test/compiler_crashers_2_fixed/0150-rdar39055736.swift

```diff
@@ -0,0 +1,17 @@
+// RUN: %target-typecheck-verify-swift %s
```

Comment:
Ah actually this has to use `mock-sdk` as an option, can you please fix?

Reply:
Yeah, sure. Like below?
```
// RUN: %target-swift-frontend(mock-sdk: %clang-importer-sdk) -typecheck %s -verify
// REQUIRES: objc_interop

import Foundation
...
```

Reply:
Yes! Sorry U didn’t notice it right away...

Reply:
No worries - thanks for reviewing it! I'll update shortly.